deploy/run.rb

```diff
@@ -21,7 +21,7 @@ def run_for_packages
 end
 
 def package_names
-  return ENV["PACKAGE_NAMES"].split(",") if ENV["PACKAGE_NAMES"]
+  return ENV["PACKAGE_NAMES"].split(",") unless ENV["PACKAGE_NAMES"]&.empty?
 
   [ENV["PACKAGE_NAME"]]
```

Comment:
100% just a personal nitpick, but I find `unless` with a negative case in a postfix hard to read.

I might recommend

```ruby
if ENV["PACKAGE_NAMES"].present?
  ENV["PACKAGE_NAMES"].split(",")
else
  [ENV["PACKAGE_NAME"]]
end
```